style.css

```diff
@@ -0,0 +1,815 @@
+	background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
+	background-size: 750px;
+	background-position: center;
+	}
+
+	.header__inner {
+		width: 100%;
+		height: 140px;
+		background: #fff;
+		z-index: 2;
+	}
+
+	.header__menu {
+		background: #EDEDED;
+		z-index: 3;
+		height: 140px;
+		width: 140px;
+	}
+
+	.header__menu-item {
+		height: 4px;
+		width: 48.7px;
+		margin-bottom: 10px;
+	}
+
+	.header__logo p {
+		margin: 25px 21px;
+		color: black;
+		font-size: 60px;
+		letter-spacing: 4px;
+	}
+
+	.text__title {
+		position: inherit;
+		align-items: flex-end;
+		font-size: 98px;
+		line-height: 103px;
+		padding-left: 52px;
+		letter-spacing: 5px;
+	}
+
+	.text__container {
+		display: flex;
+		align-items: flex-end;
+		width: 100%;
+		max-width: 750px;
+		height: 945px;
+		letter-spacing: 5px;
+	}
+
+	.step {
+		display: flex;
+	}
+
+	.text {
+		width: 100%;
+		max-width: 750px;
+		height: 600px;
+	}
+
+	.vectormob {
+			position: absolute;
+		width: 67px;
+		height: 39px;
+		left: 345px;
+		top: 1016px;
+		display: block;
+	}
+}
+
+@media (max-width: 660px) {
+	.first-container {
+		padding: 267px 0;
+		background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
+		background-size: 660px;
+		background-position: center;
+	}
+
+	.header__inner {
+		width: 100%;
+		height: 123px;
+		background: #fff;
+		z-index: 2;
+	}
+
+	.header__menu {
+		background: #EDEDED;
+		z-index: 3;
+		height: 123px;
+		width: 123px;
+	}
+
+	.header__menu-item {
+		height: 4px;
+		width: 47.7px;
+		margin-bottom: 7px;
+	}
+
+	.header__logo p {
+		margin: 22px 16px;
+		color: black;
+		font-size: 53px;
+		letter-spacing: 4px;
+	}
+
+	.text__container {
+		display: flex;
+		align-items: flex-end;
+		width: 100%;
+		max-width: 750px;
+		height: 814px;
+		letter-spacing: 5px;
+	}
+
+	.text__title {
+		position: inherit;
+		align-items: flex-end;
+		font-size: 86px;
+		line-height: 92px;
+		padding-left: 52px;
+		letter-spacing: 2px;
+	}
+
+	.vectormob {
+		position: absolute;
+		width: 65px;
+		height: 37px;
+		left: 300px;
+		top: 873px;
+		display: block;
+	}
+}
+
+@media (max-width: 565px) {
+	.first-container {
+		padding: 189px 0;
+		background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
+		background-size: 565px;
+		background-position: center;
+	}
+
+	.header__inner {
+		width: 100%;
+		height: 104px;
+		background: #fff;
+		z-index: 2;
+	}
+
+	.header__menu {
+		background: #EDEDED;
+		z-index: 3;
+		height: 104px;
+		width: 104px;
+	}
+
+	.header__menu-item {
+		height: 4px;
+		width: 42.7px;
+		margin-bottom: 7px;
+	}
+
+	.header__logo p {
+		margin: 15px 9px;
+		color: black;
+		font-size: 45px;
+		letter-spacing: 4px;
+	}
+
+	.text__container {
+		display: flex;
+		align-items: flex-end;
+		width: 100%;
+		max-width: 750px;
+		height: 677px;
+		letter-spacing: 5px;
+	}
+
+	.text__title {
+		position: inherit;
+		align-items: flex-end;
+		font-size: 73px;
+		line-height: 80px;
+		padding-left: 43px;
+		letter-spacing: 3px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)